tessellation/src/basic_shapes.rs

```diff
@@ -515,7 +515,7 @@ pub fn fill_rounded_rectangle<Output: GeometryBuilder<FillVertex>>(
         let radius = radii[i];
         if radius > 0.0 {
 
-            let arc_len = 0.5 * PI * radius;
+            let arc_len = 2.0 * PI * radius;
```

Comment:
I think that this was correct here as `0.5 * PI * radius`, since we are calling this 4 times (one for each quarter circle).